minio/exports.sh

```diff
@@ -0,0 +1,2 @@
+export APP_MINIO_ROOT_USER="minioadmin"
+export APP_MINIO_ROOT_PASSWORD="minioadmin"
```

Comment:
We can delete this exports.sh file and just place these directly in the compose file.